nucliadb/nucliadb/ingest/processing.py

```diff
@@ -227,6 +196,20 @@ async def initialize(self):
     async def finalize(self):
         await self.session.close()
 
+    @alru_cache(maxsize=None)
+    async def get_configuration(self, kbid: str) -> Optional[KBConfiguration]:
+        if self.onprem is False:
+            return None
+
+        ingest = get_ingest()
+
+        kb_obj = KnowledgeBoxID()
+        kb_obj.uuid = kbid
+        pb_response: GetConfigurationResponse = await ingest.GetConfiguration(kb_obj)
+        if pb_response.status.status == OpStatusWriter.Status.OK:
+            return KBConfiguration.from_message(pb_response.config)
+        return None
```

Comment:
I would do it the other way, to avoid having a `return None` as the last line since that's the default:

```
if pb_response.status.status!= OpStatusWriter.Status.OK:
    return None
return KBConfiguration.from_message(pb_response.config)
```